<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Open browser]]>
    </line>
    <line>
      <![CDATA[test started...]]>
    </line>
    <line>
      <![CDATA[Navigate to url]]>
    </line>
    <line>
      <![CDATA[Click on login link]]>
    </line>
    <line>
      <![CDATA[provide valid credentials]]>
    </line>
    <line>
      <![CDATA[Click on login button]]>
    </line>
    <line>
      <![CDATA[Test script failed]]>
    </line>
  </reporter-output>
  <suite started-at="2023-11-20T11:23:01 IST" name="Suite" finished-at="2023-11-20T11:23:10 IST" duration-ms="8879">
    <groups>
    </groups>
    <test started-at="2023-11-20T11:23:01 IST" name="Test" finished-at="2023-11-20T11:23:10 IST" duration-ms="8879">
      <class name="listener.TestScript">
        <test-method is-config="true" signature="browserSetup()[pri:0, instance:listener.TestScript@1bd4fdd]" started-at="2023-11-20T11:23:01 IST" name="browserSetup" finished-at="2023-11-20T11:23:04 IST" duration-ms="2903" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Open browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- browserSetup -->
        <test-method signature="login()[pri:0, instance:listener.TestScript@1bd4fdd]" started-at="2023-11-20T11:23:04 IST" name="login" finished-at="2023-11-20T11:23:10 IST" duration-ms="5746" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at listener.TestScript.login(TestScript.java:24)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[test started...]]>
            </line>
            <line>
              <![CDATA[Navigate to url]]>
            </line>
            <line>
              <![CDATA[Click on login link]]>
            </line>
            <line>
              <![CDATA[provide valid credentials]]>
            </line>
            <line>
              <![CDATA[Click on login button]]>
            </line>
            <line>
              <![CDATA[Test script failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- listener.TestScript -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
